<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PnP test application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnLoadTst">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load board image</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Origins vs geomtertic center</string>
    </property>
    <widget class="QRadioButton" name="radioOrigin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>110</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Origins</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioGeometric">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Pads geometric center</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnLoadPlacement">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load PnP Eagle data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadCCD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Take a CCD image</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1071</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Eagle placement data</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <attribute name="headerHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="treePackages">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Items for placement</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>tray 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tray 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tray 3</string>
     </property>
    </item>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblAbout">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PnP SW by Hynek Stetina
VUTBR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Placement simulator</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Board is not aligned!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAlign">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Align procedure</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgCCD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>600</width>
      <height>358</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Simulation image</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>131</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblO1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblO2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblO3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>121</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblP1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblP2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblP3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="imgCCD2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>410</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Real CCD image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPort">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>10</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>/dev/ttyACM0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMachineStatus">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>50</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine is not connected</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>140</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Board alignment</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Motion</string>
     </attribute>
     <widget class="QPushButton" name="btnMoveXp">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>20</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>→</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveXn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>20</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveYp">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveYn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveXn2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>⇇</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveXp2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>⇉</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveYp2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>39</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>⇈</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveYn_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>⇊</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCenter">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>99</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHome">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Home all</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveZp">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Z ↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveZn">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>113</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Z ↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveRn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>↶ R</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveRp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>R ↷</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QPushButton" name="btnSaveSettings">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply settings</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox#groupBox_2 { 
     border: 2px solid gray; 
     border-radius: 3px; 
 }
QGroupBox::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:0 10px;
     border: 2px solid gray; 
 } </string>
      </property>
      <property name="title">
       <string>Board offset</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBoardY">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>113</width>
         <height>33</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBoardX">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>113</width>
         <height>33</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCurrentPos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Get current position</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1341</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
